python/lsst/ts/rubintv/handlers/api.py

```diff
@@ -76,6 +77,23 @@ async def redis_post(request: Request, message: KeyValue) -> dict:
         return {"response": response}
 
 
+@api_router.get("/slac", response_class=RedirectResponse)
+async def redirect_slac_no_slash(request: Request) -> RedirectResponse:
+    """Redirects ``/rubintv/slac`` to ``/rubintv/usdf``."""
+    # redirect to the new path
+    new_url = request.url.replace(path="/rubintv/usdf")
+    return RedirectResponse(url=str(new_url), status_code=301)
+
+
+@api_router.get("/slac/{path:path}", response_class=RedirectResponse)
+async def redirect_slac(path: str | None, request: Request) -> RedirectResponse:
+    old_path = request.url.path
+    # only replace the first occurrence
```

Comment:
Maybe just remove this comment line and docstrings as you did with `redirect_slac_no_slash`?